<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSpaceObject</class>
 <widget class="QDialog" name="AddSpaceObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add fake space object</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>241</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Star</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Planet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Satellite</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Object's Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Object's Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_o">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create your own fake space object</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>121</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>121</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>111</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit_10"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_12"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddSpaceObject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddSpaceObject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
